yugabyte/src/yugabyte/ycql/multi_key_acid.clj

```diff
@@ -1,4 +1,7 @@
 (ns yugabyte.ycql.multi-key-acid
+  "Given a single table of two-column composite key and one value column,
+  execute reads and transactional batches of writes.
+  Verify that history remains linearizable."
```

Comment:
Do we need to duplicate test description in both of its client? It might be enough to have a description in `yugabyte/src/yugabyte/<test>.clj` ?

Reply:
Removed test descriptions from api-specific test code, left it near the workload